lib/helpers.js

```diff
@@ -850,22 +850,9 @@ function toAvdLocaleArgs (language, country) {
 async function getAndroidPrefsRoot () {
   // https://gerrit.pixelexperience.org/plugins/gitiles/development/+/5c11852110eeb03dc5a69111354b383f98d15336/tools/androidprefs/src/com/android/prefs/AndroidLocation.java
```

Comment:
it makes sense to update this link to explain the actual implementation

Reply:
Removed the link